dev-util/roctracer/roctracer-6.1.1.ebuild

```diff
@@ -67,6 +67,11 @@ src_prepare() {
 }
 
 src_configure() {
+	rocm_use_hipcc
```

Comment:
If I remember correctly roctracer can be built with gcc, since it is a host library, not containing GPU kernel.

Only when building with `test`, it compiles some hip program (to test the tracing capabilities). So my humble suggestions are (in order of ascending gracefulness):

1. `rocm_use_hipcc` only when using `test`, or
2. just use `strip-unsupported-flags` when using `test`, or
3. separate the build process of test suite, like `dev-util/hip`, but that require some patch to `test/CMakeLists.txt `

Reply:
`rocm_use_hipcc` only when using `test` sounds not great: this technically means that emerge without tests installs completely different shared library (different size, different methods of hardened flags, maybe libstdc++ instead of libc++, potential unidentified problems if library used clang-specific code and developers never tested gcc builds due to hipcc requirement). I suggest to keep it simple/stupid; anyways, hipcc will very likely be installed by other packages.

Reply:
>  I suggest to keep it simple/stupid; anyways, hipcc will very likely be installed by other packages.

OK, that seems reasonable. I'll run tests to see if switching to hipcc/clang cause any issues